mods/d2k/sequences/infantry.yaml

```diff
@@ -83,7 +83,6 @@ light_inf:
 		Filename: DATA.R16
 		Start: 4272
 		Offset: -30,-24
-		-Remap:
```

Comment:
So change this to `Remap: 00000000` instead to fix #21444?

Reply:
Sure, pushed a separate commit. It does indeed fix the issue.